CODEOWNERS

```diff
@@ -9,9 +9,51 @@
 #
 #
 # Learn about CODEOWNERS file format:
-#  https://help.github.com/en/articles/about-code-owners
+# https://help.github.com/en/articles/about-code-owners
 #
+# NOTE: Lines should be entered in the following format:
+# <component_path_relative_from_project_root>/<min_1_space><owner_1><space><owner_2><space>..<owner_n>
+# instrumentation/net/http/otelhttp/         @open-telemetry/collector-go-approvers @madvikinggod @mralias
+# Path separator and minimum of 1 space between component path and owners is
+# important for validation steps
+# 
 
-* @jmacd @MrAlias @Aneurysm9 @evantorrie @XSAM @dashpole @MadVikingGod @pellared @hanyuancheung @dmathieu
+* @open-telemetry/go-approvers
 
 CODEOWNERS @MrAlias @Aneurysm9 @MadVikingGod
```

Comment:
I left this, but it might make it so others can't make a PR aginst this file.  If someone (who isn't me because I'm in the list) could try I would appreciate it.